package/MDAnalysis/topology/ITPParser.py

```diff
@@ -27,7 +27,7 @@
 
 Reads a GROMACS ITP_ or TOP_ file to build the system. The topology will
 contain atom IDs, segids, residue IDs, residue names, atom names, atom types,
-charges, chargegroups, masses (guessed if not found), moltypes, and molnums. 
+charges, chargegroups, moltypes, and molnums.
```

Comment:
Could you please update this slightly to be clear that if masses are found, they will be read from file?